projects/opendr_ws/src/perception/scripts/object_detection_2d_gem.py

```diff
@@ -176,25 +221,25 @@ def callback(self, msg_rgb, msg_ir):
             # e.g., opendr_detection = self.bridge.from_ros_bounding_box_list(ros_detection)
 
         if self.rgb_publisher is not None:
-            plot_rgb = draw(image_rgb, boxes, w_sensor1, mean_fps)
+            plot_rgb = draw_bounding_boxes(image_rgb, boxes, class_names=self.classes)
             message = self.bridge.to_ros_image(Image(np.uint8(plot_rgb)))
             self.rgb_publisher.publish(message)
         if self.ir_publisher is not None:
-            plot_ir = draw(image_ir, boxes, w_sensor1, mean_fps)
+            plot_ir = draw_bounding_boxes(image_ir, boxes, class_names=self.classes)
             message = self.bridge.to_ros_image(Image(np.uint8(plot_ir)))
             self.ir_publisher.publish(message)
 
 
-if __name__ == '__main__':
+if __name__ == "__main__":
     # Select the device for running the
     try:
         if torch.cuda.is_available():
             print("GPU found.")
-            device = 'cuda'
+            device = "cuda"
         else:
             print("GPU not found. Using CPU instead.")
-            device = 'cpu'
+            device = "cpu"
     except:
-        device = 'cpu'
+        device = "cpu"
     detection_estimation_node = GemNode(device=device)
     detection_estimation_node.listen()
```

Comment:
Main should have the same argparse logic as in the ROS2 node, so as to enable users to modify the behaviour of the node.